build/common.sh

```diff
@@ -875,7 +875,10 @@ function kube::release::write_addon_docker_images_for_server() {
         kube::log::status "Pulling and writing Docker image for addon: ${addon_path}"
 
         local dest_name="${addon_path//\//\~}"
-        "${DOCKER[@]}" pull "${addon_path}"
+        if [[ ! `${DOCKER[@]} images | awk '{print ($1":"$2)}' | grep ${addon_path}` ]]; then
+          echo "Addon image does not exist, pull for it: ${addon_path}"
+          "${DOCKER[@]}" pull "${addon_path}"
+        fi
         "${DOCKER[@]}" save "${addon_path}" > "${1}/${dest_name}.tar"
```

Comment:
It should already be handled by https://github.com/kubernetes/kubernetes/blob/master/build/common.sh#L893.